### ISSUE TITLE
v1.11.0+ breaks aliased dynamic imports

### ISSUE THREAD

**Issue by boris-petrov** [ef4/ember-auto-import#368]
Say I have:

```javascript
const app = new EmberApp(defaults, {
  autoImport: {
    alias: {
      'bpmn-js': 'bpmn-js/lib/Modeler.js',
    },
  },
});
```

And then a dynamic import in the code:

```javascript
const css = await import('bpmn-js/dist/assets/diagram-js.css');
```

This used to work in 1.10.1 but 1.11.0 and 1.11.1 break with:

```
Build Error (Bundler)

Can't resolve 'bpmn-js/lib/Modeler.js/dist/assets/diagram-js.css' in '/home/boris/code/project'
```

**Comment by ef4** [comment]
I We were [trying to improve `alias`](https://github.com/ef4/ember-auto-import/pull/334/files) so it works the same way that webpack's `resolve.alias` works, which does a prefix match, which means your css import is becoming `bpmn-js/lib/Modeler.js/dist/assets/diagram-js.css`.

We might need to put that behavior behind a new option, at least until we do a semver major release.

**Comment by boris-petrov** [comment]
What's webpack's way of doing what I'm trying to achieve?

**Comment by ef4** [comment]
They offer a trailing `$` for exact matches. But our implementation doesn't have that. It really seems not ready to roll out, I'm reverting it here: https://github.com/ef4/ember-auto-import/pull/369

**Comment by ef4** [comment]
Fixed in 1.11.2.
